modules/extensions-spring-support/src/test/java/org/mule/test/module/extension/parameter/value/ParameterWhitespaceTrimmingTestCase.java

```diff
@@ -129,7 +143,7 @@ public void whitespacesAreNotTrimmedForPojoTextExpression() throws Exception {
   @Test
   public void whitespacesAreTrimmedForPojoTextCDATA() throws Exception {
```

Comment:
I think that the name of the test should be changed to whitespacesAreNotTrimmedForPojoTextCDATA if cdata is not going to be trimmed.